core/templates/dev/head/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -40,6 +40,9 @@ oppia.controller('SettingsTab', [
       EXPLORATION_TITLE_INPUT_FOCUS_LABEL);
 
     $scope.CATEGORY_LIST_FOR_SELECT2 = [];
+    //Empty category added to force dropdown to scroll top
```

Comment:
Hi @szangaro -- please go ahead and make a PR with whichever version you think best; it's easy to change later if needed, but that allows folks to play with the current version in the meantime. (I suspect @souravbadami is currently asleep!)